src/ec/packages/ec-component-timeline/timeline.story.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable no-param-reassign */
+
+import { storiesOf } from '@storybook/html';
+import { withKnobs, number } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+import iconPath from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+
+import demoData from './demo/data';
+
+import timeline from './timeline.html.twig';
+import notes from './docs/timeline.md';
+
+storiesOf('Components/Timeline', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withCode)
+  .addDecorator(withNotes)
+  .add(
+    'default',
+    () => {
+      const from = number('From', 10);
```

Comment:
what about using `demoData.hide.from` as a default instead of 10? (same for `to`) 
thus the changes made to `demo/data.js` would be reflected here as well